docs/source/user_guide/model_understanding.ipynb

```diff
@@ -448,8 +448,18 @@
    "source": [
     "from evalml.model_understanding.prediction_explanations import explain_predictions\n",
     "\n",
-    "report = explain_predictions(pipeline=pipeline, input_features=X.iloc[[0, 4, 9]], include_shap_values=True)\n",
-    "print(report)"
+    "report = explain_predictions(pipeline=pipeline, input_features=X.iloc[[0, 4, 9]], include_shap_values=True,\n",
+    "                            output_format='dataframe')\n",
```

Comment:
@freddyaboulton could you explain how to read the output here?

![Screen Shot 2021-02-03 at 12 26 19 PM](https://user-images.githubusercontent.com/235870/106785135-13f67080-661b-11eb-87ae-c5a5059f64cd.png)

Each grouping of rows corresponds to a different prediction explanation, yes? And that's indicated by the "rank" column. What confuses me is that "rank" ranges from 0 to n-1 whereas the indices you passed to `input_features` are 0, 4 and 9. I'm concerned someone looking at the dataframe output will have trouble matching those row groups up with the original dataframe row that info corresponds to.

If I'm following correctly here, I suggest we either find a way to get the original dataframe index into this dataframe report, or we change our language a bit and call this "prediction number" instead of "rank". The latter option seems more appealing.

What do you think?

Reply:
@dsherry You're spot on about the grouping! I agree with the rename of `rank` to `prediction_number`. Adding the original dataframe index is possible but we don't do that in `explain_prediction` for the other output formats.

Reply:
![image](https://user-images.githubusercontent.com/41651716/106790628-d0533500-6621-11eb-93d9-47485beb0e2a.png)